connector.go

```diff
@@ -125,8 +125,13 @@ type Connector interface {
 
 	// DDL operations (schema)
 	// CheckSchema validates that the set of entities you have provided is valid and registered already
-	// It returns a list of SchemaRef objects for use with later DML operations.
-	CheckSchema(ctx context.Context, scope string, namePrefix string, ed []*EntityDefinition) (version int32, err error)
+	// It returns the latest schema version for use with later DML operations.
+	CheckSchema(ctx context.Context, scope string, namePrefix string, eds []*EntityDefinition) (version int32, err error)
+	// CanUpsertSchema is used by the CLI to validate whether new entities
```

Comment:
nit: don't mention the CLI here. users of the function may change over time.

Reply:
removed.